openapi.yml

```diff
@@ -238,36 +238,20 @@ paths:
         - Threads
```

Comment:
The whole of this document isn't quite right, formatting is wrong and example are incorrect. I will raise a card to tackle everything